### PR TITLE
[DC-1804] Update imports in controlled_tier_qc module to absolute paths so it can be invoked from any location

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/controlled_tier_qc/code/config.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
 from pathlib import Path
 
-CSV_FOLDER = Path('csv')
-SQL_FOLDER = Path('sql')
+CSV_FOLDER = Path('analytics/cdr_ops/controlled_tier_qc/CSV')
```
Comment (nishanthpp93) [inline data_steward/analytics/cdr_ops/controlled_tier_qc/code/config.py:3]: Should this be `csv`?

**Review by nishanthpp93 — COMMENTED** [review]
Looks good. I assume all the other changes are due to `yapf`?

File: data_steward/analytics/cdr_ops/controlled_tier_qc/code/config.py (hunk 1 above)
Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/controlled_tier_qc/code/config.py:3]: you are right it should be 'csv'. I'll update it.

Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/controlled_tier_qc/code/config.py:3]: And yes, other changes were due to YAPF.

**Review by nishanthpp93 — APPROVED** [review]
Looks good!

**Pull request merged**